src/styles/dropdown.css

```diff
@@ -1 +1,78 @@
 /* dropdown */
+
+.ui.selection.dropdown {
+  border-radius: 4px;
+  border: 1px solid var(--neutral-900);
+  padding: var(--spacing-1) var(--spacing-4) var(--spacing-1) var(--spacing-2);
+  min-width: 0;
+  min-height: 0;
+  font-family: var(--gotham);
+}
+
+.ui.selection.dropdown .divider.text {
+  color: var(--shade-black);
+  line-height: 1.5rem;
+}
+
+.ui.selection.dropdown .divider.default.text {
+  color: var(--neutral-900);
+}
+
+/** Focus **/
+.ui.selection.dropdown:not(.error):focus {
+  border-color: var(--primary-pms-362-900);
+}
+
+/** Focus and Hover **/
+.ui.selection.dropdown:not(.error):focus:hover {
+  border-color: var(--primary-pms-362-800);
+}
+
+/** Hover **/
+.ui.selection.dropdown:not(.error):hover {
+  border-color: var(--primary-pms-362-800);
+}
+
+/** Active **/
+.ui.active.visible.selection.dropdown:not(.error) {
+  border-color: var(--primary-pms-362-900);
+}
+
+/** Error **/
+.ui.selection.dropdown.error {
+  border: 2px solid var(--alert-error-900);
```

Comment:
This border should just be `1px`.

Reply:
I believe the designs show it as 2px:

<img width="977" alt="image" src="https://github.com/abby-moffitt/GA-Semantic/assets/101662434/01a7c999-6caf-4ba4-b91f-b4596037580e">

Reply:
Oh, I stand corrected, though I still think we should use 1px. Otherwise, the dropdown might jump a pixel when an error is thrown (this must be why we used 1px in our library).

Reply:
Got it! I will change it then!